### PR TITLE
Setup readme.md for publishing

### PR DESCRIPTION
**Pull request by neriyarden**
@unicop you can take a look and give your feedback.

also, how much documentation do want to put in the readme file?

### CODE REVIEW

**Comment by neriyarden** [comment]
[[README] Setup readme properly for publish, including logos, intro, description, etc. (Use other open-sources as refs)](https://trello.com/c/XeK2CLWi/23-readme-setup-readme-properly-for-publish-including-logos-intro-description-etc-use-other-open-sources-as-refs)

File: README.md [hunk 1]
```diff
@@ -1,14 +1,61 @@
-# ESLint Plugin for Migrate To TypeScript
+<h1 align="center">Migrate To Typescript</h1>
```
Comment (unicop) [inline README.md:1]: we will change in the future when we will make it mono-repo and moving this readme to the relevant package dir

File: README.md [hunk 2]
```diff
@@ -1,14 +1,61 @@
...
+
+
+<br/>
+
+
+
+## Introduction
+This eslint plugin is a collection of custom eslint rules, as well as some cherry-picked rules from the most popular and reputable eslint plugins out there:
+
+ - [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint)
```
Comment (unicop) [inline README.md:38]: so this section is incorrect at the moment, we configured ourselves these plugins, but we are not exposing them at the moment, we will have to think about how we should do it!

File: README.md [hunk 3]
```diff
@@ -1,14 +1,61 @@
...
+ - [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier)
+ - [eslint-plugin-promise](https://github.com/xjamundx/eslint-plugin-promise)
+ - [eslint-plugin-security](https://github.com/nodesecurity/eslint-plugin-security)
+ - [eslint-plugin-unicorn](https://github.com/sindresorhus/eslint-plugin-unicorn)
+
+
+## Install
+
+```sh
+npm install --save-dev eslint-plugin-migrate-to-typescript
```
Comment (unicop) [inline README.md:50]: we will have docs, we should direct to them, the readme is just an introduction and why people should use it

**Review by unicop — APPROVED** [review]
see comments below, in general looks good for now :) 
it will be changed a lot!

**Review by unicop — APPROVED** [review]
good for now

**Pull request merged**